app/weeklyPicks/WeeklyPicks.tsx

```diff
@@ -100,6 +100,9 @@ const WeeklyPicks = ({ NFLTeams, currentGameWeek }: Props): JSX.Element => {
 
     if (!gameGroupData || !weeklyPicksData) {
       console.error('Error getting game data');
+      toast.custom(
+        <Alert variant={AlertVariants.Error} message="Something went wrong." />,
+      );
       return;
     }
```

Comment:
I was curious in line 174, could we have `You picked {currentUserPick[user.id].team} was successful` => `You picked Falcons was successful`. Thoughts? :)

Reply:
i definitely can! I'll give that a try so people know for sure that the system saw what they picked.